okta-hosted-login/server.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const session = require('express-session');
+const cons = require('consolidate');
+const path = require('path');
+const { ExpressOIDC } = require('@okta/oidc-middleware');
+
+const config = require('./.samples.config.json').oktaSample;
+const templateDir = path.join(__dirname, 'views');
+const frontendDir = path.join(__dirname, 'assets');
+
+
+const oidc = new ExpressOIDC({
+  issuer: config.oidc.issuer,
+  client_id: config.oidc.clientId,
+  client_secret: config.oidc.clientSecret,
+  redirect_uri: config.oidc.redirectUri,
+  scope: config.oidc.scope
+});
+
+const app = express();
+
+app.use(session({
+  secret: 'this-should-be-very-random',
+  resave: true,
+  saveUninitialized: false
+}));
+
+// Provide the configuration to the view layer, as we show it on the homepage
+const displayConfig = Object.assign({}, config.oidc, { clientSecret: '****' + config.oidc.clientSecret.substr(config.oidc.clientSecret.length - 4, 4)});
+
+app.locals.oidcConfig = displayConfig;
+
+app.use('/assets', express.static(frontendDir));
```

Comment:
nit: Add a comment above here explaining how this is specifying our template engine + where our assets are located.